app/assets/stylesheets/_session.scss

```diff
@@ -1,17 +1,17 @@
 @import 'base';
 
 #login-form{
-  width:60%;
+  width:366px;
```

Comment:
Properties should be ordered background, border, display, margin, padding, text-align, width<br>Colon after property should be followed by one space